docs/encryption.md

```diff
@@ -56,10 +56,14 @@ setopt HIST_IGNORE_SPACE
 alias jrnl=" jrnl"
 ```
 
-The fish shell does not support automatically preventing logging like
-this. To prevent `jrnl` commands being logged by fish, you must make
-sure to type a space before every `jrnl` command you enter. To delete
-existing `jrnl` commands from fish’s history, run
+If you are using `fish` instead of `bash` or `zsh`, you can get the same behaviour by
+adding this to your `fish` configuration:
+
+``` sh
+abbr jrnl=" jrnl"
```

Comment:
I am terribly sorry but there is a typo here. It should be `abbr jrnl " jrnl"`. How does one fix that now that this pull request has been merged?